spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -102,7 +102,7 @@ public SubscriberStub createSubscriberStub(RetrySettings retrySettings) {
 			SubscriptionAdminSettings.Builder subscriptionAdminSettingsBuilder = SubscriptionAdminSettings.newBuilder()
 					.setExecutorProvider(this.executorProvider)
 					.setCredentialsProvider(this.credentialsProvider)
-					.setTransportChannelProvider(this.channelProvider);
```

Comment:
Has `setTransportChannelProvider` been removed??

Reply:
The only thing we were using it for was to set the similar header the way we were doing before (which was useless).
Not to worry because #280 is bringing all that stuff back.